app/services/collection_version_event_description_builder.rb

```diff
@@ -16,12 +16,14 @@ def build
     [name, description, contact_email, related_links].compact.join(", ")
   end
 
+  private
+
   def name
     "collection name modified" if form.changed?(:name)
   end
 
   def description
-    "description modified" if form.changed?("description")
+    "description modified" if form.changed?(:description)
   end
```

Comment:
Opportunistic changes for consistency (use symbols for form field checks) and clarity of contract (make non-public methods explicitly private.